### ISSUE TITLE
https://raw.githubusercontent.com/riswinto/SC-400T00A-Microsoft-Information-Protection-Administrator/master/Instructions/Labs/LAB_AK_02_Lab1_Ex2_Endpoint_DLP.md

### ISSUE THREAD

**Issue by TesterTesterson1004** [MicrosoftLearning/SC-400T00A-Microsoft-Information-Protection-Administrator#184]
# Module: 0?
#### Step: 05

Description of issue

<img width="191" alt="image" src="https://github.com/user-attachments/assets/a566c941-fde9-4161-890e-6bce835769cd">

This step seems to be a duplicate of the previous step: Per this screenshot from just now, in case the UI has changed since release, right after selecting Settings/Device onboarding/Onboarding, we end up on this page:

<img width="590" alt="image" src="https://github.com/user-attachments/assets/424315a3-9387-4a44-8183-1c7df1f07424">

After Step 4, we end up on the 'Onboarding' page, and thus Step 5 could be deleted.  Then, if we jump right to Step 6, phrasing could be cleaned up to be:

_"...On the Onboarding page, In the Deployment method dropdown menu, select Local Script (for up to 10 machines) and select Download package..."_

**Comment by riswinto** [comment]
Thank you for the feedback and suggestion. These steps have been updated as suggested.
